src/types/json.h

```diff
@@ -65,6 +66,29 @@ struct JsonValue {
     return Status::OK();
   }
 
+  Status Append(std::string_view path, const std::string &append_value, std::vector<uint64_t> &append_cnt) {
+    try {
+      auto append_trimmed = util::Trim(append_value, "\"");
+      jsoncons::jsonpath::json_replace(value, path, [&append_trimmed, &append_cnt](const std::string & /*path*/, jsoncons::json &origin) {
+        uint64_t len = 0;
+        if (origin.is_string()) {
+          auto origin_str = origin.to_string();
+          // need trim "
+          auto origin_trimmed = util::Trim(origin_str, "\"");
+          len = origin_trimmed.length() + append_trimmed.length();
+          std::string new_value;
+          new_value.append("\"").append(origin_trimmed).append(append_trimmed).append("\"");
+          origin = jsoncons::json::parse(new_value);
+        }
+        append_cnt.push_back(len);
```

Comment:
```suggestion
        } else {
          append_cnt.push_back(0);
        }
```

Reply:
`len` initialization 0，so there logic is right

Reply:
Yes, but use `0` should be more intuitive, we don't care how the length was initiailized.

Reply:
ok, I understand